### PR TITLE
Release 1.0.0

### PR DESCRIPTION
**Pull request by timkpaine**

### CODE REVIEW

File: setup.py
Comment (mgaitan) [inline setup.py:1]: can't we merge this into pyproject and remove setup.py? and please mention yourself as maintainer

Comment (timkpaine) [reply setup.py:1]: we can move to pyproject.toml, though some reorg is necessary of files (e.g. the long_description building right now)

Comment (mgaitan) [reply setup.py:1]: I see. we should simplify the readme and move the current as  the doc index
then we could add urls in the toml 

```
[project.urls]
"documentation" = "...."
"Changelog" = "https://github.com/mgaitan/sphinxcontrib-mermaid/blob/master/CHANGELOG.rst"
```

Comment (mgaitan) [inline setup.py:1]: `python_requires=">=3.8",`  we should upgrade this

Comment (jensens) [reply setup.py:1]: > can't we merge this into pyproject and remove setup.py? 

IMO we should  remove it. 
We should be aware of the side effects in the sphinxcontrib namespace when we [mix PEP 420 Python packages with old style packages](https://yenzenz.com/blog/2023/mixed_python_pep420/) which is a problem only when installing as editables.

Also, I am not sure how Sphinx internal discovery of extension packages works and if this needs special attention?

Comment (timkpaine) [reply setup.py:1]: I think we're good on both points

**Comment by jensens** [comment]
So, I added the branch to two setups with git+https in requirements.txt and it works. Are there special combinations I should test additionally?

**Review by jensens — APPROVED** [review]

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,23 @@
 Changelog
 ---------
 
+1.0.0 (September 19, 2024)
```
Comment (mgaitan) [inline CHANGELOG.rst:4]: remember to update the date

**Review by mgaitan — APPROVED** [review]

**Comment by mgaitan** [comment]
Thanks guys!

**Comment by AA-Turner** [comment]
FYI - https://github.com/sphinx-doc/sphinx/issues/13002

**Comment by Paebbels** [comment]
What's missing to get this release done?  
Is there a timeline for the release?

------------
Currently the problem reported at #160/#161 are blocking my upgrades to Sphinx 8.1 in many projects.

**Comment by michaelosthege** [comment]
who knows how many docs builds are failing at the moment - @mgaitan @timkpaine a release would be awesome!

**Pull request merged**